src/sentry/snuba/events.py

```diff
@@ -211,18 +212,41 @@ class Columns(Enum):
     )
 
 
+class BaseColumn(object):
+    def get(self):
+        raise NotImplementedError
+
+
+class BuiltInColumn(BaseColumn):
+    def __init__(self, col):
+        self.__col = col
+
+    def get(self, dataset=Dataset.Discover):
```

Comment:
dataset parameter is intended to be temporary, since we have not fully migrated to the Discover dataset

Reply:
If this lands before we move over to the transactions dataset you'll likely need a case for the transactions dataset too.

Reply:
I think I willfully forgot about that one. Added it now.